src/components/DataTable/defaultSelectAllRenderer.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export default ({ allSelected, dataKey, onSelectAll, tableId }) => { // eslint-disable-line react/prop-types
+  const checkboxId = `${tableId}-${dataKey}`;
+
+  return (
+    <th
+      className="slds-text-align--right"
+      key={dataKey}
+      scope="col"
+      style={{ width: '3.25rem' }}
+    >
+      <div>
+        <span className="slds-checkbox">
+          <input
```

Comment:
This replicates `CheckbowRaw`, is there a reason it is not used here?

Reply:
No, will change 👍